.github/workflows/test.yml

```diff
@@ -199,7 +199,7 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y graphviz pkg-config libgraphviz-dev
-          pip install pygraphviz
+          pip install pygraphviz==1.7.*
```

Comment:
Instead of doing this, why don't we do
```
pip install -e .[docs,graph]
```
on L206?

Reply:
No reason, probably an omission of mine.

Reply:
@wxtim could you remove this line and add the optional output `graph` to the `pip install -e .[docs]` line below. That way it will use the pygraphviz version range specified in the `setup.cfg` file.